sdk/contentsafety/azure-ai-contentsafety/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0 (Unreleased)
 
 ### Features Added
-
+- Support AAD Authentication
+- Support 8 severity level for text:analyze API
 ### Breaking Changes
-
+- Contract change for AnalyzeText, AnalyzeImage, Blocklist management related methods
 ### Bugs Fixed
 
 ### Other Changes
```

Comment:
Something like this

https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md?plain=1#L17-L20

under "Other Changes"

Reply:
added dependency package changes